### PR TITLE
feat: add wear os template to the create command

### LINKED ISSUE #689: feat: add very good wear app template
**Description**

Allow people to create wear apps via the `create` command.

Proposed syntax:
```
very_good create flutter_app --template=wear my_wear_app
```

**Additional Context**
https://github.com/VeryGoodOpenSource/very_good_wear_app

### PR DESCRIPTION
**Pull request by renancaraujo**
## Description

feat: add wear os template to the create command

fix #689

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/src/commands/create/commands/flutter_app.dart [hunk 1]
```diff
@@ -39,4 +36,13 @@ class CreateFlutterApp extends CreateSubCommand with OrgName {
 
     return vars;
   }
+
+  @override
+  String get defaultTemplateName => 'core';
```
Comment (wolfenrain) [inline lib/src/commands/create/commands/flutter_app.dart:41]:
```suggestion
  String get defaultTemplateName => templates.first.name;
```

Can't this be the default either? What if we change the template name and forget it here for instance?

Comment (renancaraujo) [reply lib/src/commands/create/commands/flutter_app.dart:41]: >  What if we change the template name and forget it here for instance?

Then we will have broken unit tests

Comment (wolfenrain) [reply lib/src/commands/create/commands/flutter_app.dart:41]: Good point! But imho then we should at least use the reference like this, that way we dont have to think about it anyway :D

Comment (renancaraujo) [reply lib/src/commands/create/commands/flutter_app.dart]: Yeah, changed to be the default behavior fo the mixin

**Review by wolfenrain — APPROVED** [review]

**Pull request merged**